contrib/win32/win32compat/termio.c

```diff
@@ -52,22 +52,15 @@
 extern int in_raw_mode;
 BOOL isFirstTime = TRUE;
 
-struct io_status {
-	DWORD to_transfer;
-	DWORD transferred;
-	DWORD error;
-};
-static struct io_status read_status, write_status;
-
 /* APC that gets queued on main thread when a sync Read completes on worker thread */
 static VOID CALLBACK
 ReadAPCProc(_In_ ULONG_PTR dwParam)
 {
 	struct w32_io* pio = (struct w32_io*)dwParam;
 	debug5("TermRead CB - io:%p, bytes: %d, pending: %d, error: %d", pio, read_status.transferred,
 		pio->read_details.pending, read_status.error);
```

Comment:
cleanup the static instance of read_status & write_status in this file?

Reply:
don't understand. they are cleaned up as indicated in PR description.